omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResource.java

```diff
@@ -85,16 +89,16 @@ public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoe
         description.addProperty("description");
         return description;
     }
-
-    @PropertyGetter("identifierType")
-    public String getidentifierType(IdentifierSource identifierSource) {
-        return identifierSource.getIdentifierType().getUuid();
-    }
 	
     @PropertyGetter("display")
     public String getDisplayString(IdentifierSource identifierSource) {
         return identifierSource.getIdentifierType() + " - " + identifierSource.getName();
     }
+    
+    @PropertyGetter("sourceType")
```

Comment:
Property getters are supposed to be used for properties that are not part of the resource object.

Reply:
the sourceType is not from the database. It tells the user which class the identifier source belongs to.

Reply:
@cnagadya seems to have a strong point. You do not need to add a property to the resource for just this. All you need is a parameter which can have those three values.

Reply:
ok. will add the class to the display property.